express-zod-api/tests/__snapshots__/documentation.spec.ts.snap

```diff
@@ -1365,6 +1365,18 @@ components:
       required:
         - name
         - subcategories
+    Schema3:
```

Comment:
now need to figure this out

Reply:
this is because we run `depict()` in `depictRequestParams` now, unconditionally.
while also keep running it in `depictBody`.
perhaps it's time for them to use same grounds

Reply:
fixed in b7b0309e